addons/mail/static/src/models/activity_menu_view.js

```diff
@@ -79,19 +79,14 @@ registerModel({
         _computeCounter() {
             return this.activityGroups.reduce((total, group) => total + group.total_count, this.extraCount);
         },
-        /**
-         * @private
-         * @returns {Array[]}
-         */
-        _sortActivityGroups() {
-            return [
-                ['smaller-first', 'irModel.id'],
-            ];
-        },
     },
     fields: {
         activityGroups: many('ActivityGroup', {
-            sort: '_sortActivityGroups',
+            sort() {
+                return [
+                    ['smaller-first', 'irModel.id']
+                ];
```

Comment:
if you were going to inline arrow function, could at least keep a single line return in this case